azurerm/data_source_arm_container_registry.go

```diff
@@ -0,0 +1,87 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmContainerRegistry() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmContainerRegistryRead,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		MigrateState:  resourceAzureRMContainerRegistryMigrateState,
+		SchemaVersion: 2,
```

Comment:
we should be able to remove the Importer -> Schema version lines here, since they're not needed for data sources